iidm/iidm-api/src/main/java/com/powsybl/iidm/network/NetworkFactory.java

```diff
@@ -47,7 +48,11 @@ static NetworkFactory find(String name) {
      * @return default {@code NetworkFactory} instance.
      */
     static NetworkFactory findDefault() {
-        return find(NetworkFactoryConstants.DEFAULT);
+        String defaultName = PlatformConfig.defaultConfig()
+                .getOptionalModuleConfig("iidm")
+                .map(module -> module.getStringProperty("default"))
```

Comment:
Should you name the property `default`? Isn't `impl` more understandable for example?

Reply:
I chose this to be consistent with the loadflow.default configuration

Reply:
As we known that IIDM "branding" should one day disappear and just call it "the powsybl network model", I would have named it:

```yaml
network:
    default-impl-name: xxx
```

Reply:
```yaml
network-model:
   default: xxx
```
?

If we use default-impl-name, we should update the loadflow.default config for consistency

Reply:
I actually prefer @geofjamg's proposition, given than we will then update the `loadflow` config names to be consistent.